commit e20940f58b998f8eb35e69f73738b88fc267b48f (refs/changes/17/3717/2)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-03-26T18:04:52+00:00 (1 year, 6 months ago)
    
    OS-6823 need uefi-csm-rom.bin with fix for OS-6604
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
index 02f11ea5..0cbe839c 100755
Binary files a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin differ
